datafu-spark/src/test/resources/META-INF/services/datafu.spark.PythonResource

```diff
@@ -0,0 +1,20 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+datafu.spark.ExampleFiles
+datafu.spark.Py4JResource
+datafu.spark.PysparkResource
```

Comment:
Do we need to update the license file even for things that are only in our tests, not in our "real" jar?

The py4j zips I took as is - but I had to manually create the pyspark zips by cutting them out of the Spark source code (making a zip from the pyspark directory from the relevant Spark branch/tag) I suspect this will make using/reusing these files difficult. But I will look into adding py4j as a tests Gradle dependency.

Reply:
I didn't see a way to use standard Gradle dependencies, but I've pushed a fix that downloads pyspark/py4j and puts the prepared versions in the data directory. I also removed the zip files from our sources (although they will remain in our repository's Git history)

Reply:
Thanks for fixing that.  The license file needs to reflect what we ship in the zip artifact for our source release (is my understanding).  So if we include the py4j zip in our source zip for testing then we need to have the license reflect this.

Reply:
I think we're good, because the source artifact doesn't include these zips - they're downloaded on demand when the Gradle test or eclipse tasks are run. (I created the zip for the source release to verify this). So we can leave the license as it is.